### DISCUSSION TITLE
NequIP and pair NequIP versions

### DISCUSSION THREAD

**Discussion by youngjae142** [mir-group/pair_nequip#13] [Q&A] [answered]
Hello!
Is it recommended to use the same version for NequIP and Pair NequIP?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @ssrokyz ,

It is recommended to use the same "major" (informal backward compatible version, and latest given that), unless otherwise noted. Meaning:
0.5.* with 0.5.*
0.6.* with 0.6.* (whenever that happens)

Otherwise, unless otherwise noted, latest version should be good.

**Reply by youngjae142** [reply]
Thx for the clear answer!
